salt/metalk8s/orchestrate/downgrade/init.sls

```diff
@@ -111,6 +111,27 @@ Deploy Kubernetes service config objects:
   - require_in:
     - salt: Deploy Kubernetes objects
 
+{#- Due to a change of Loki StatefulSet labelSelector in 124.1.0, which is immutable field
+    Manually delete the Loki StatefulSet object if dest_version < 124.1.0
+    NOTE: This logic can be removed in `development/126.0` #}
+{%- if salt.pkg.version_cmp(dest_version, '124.1.0') == -1 %}
```

Comment:
I guess we could also check if the current version of the statefulset is higher or equal to 124.1 (in case we are re-running the orchestrate), but it's fine, downgrade doesn't need to be super optimized IMO

Reply:
Yes we could, but anyway the state is idempotent so it will be just do nothing since the object does not exists